moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1773,7 +1757,7 @@ bool PlanningScene::processCollisionObjectAdd(const moveit_msgs::CollisionObject
     setObjectType(object.id, object.type);
 
   // Add subframes
```

Comment:
```suggestion
  // Add subframes to the newly created or (possibly) modified object
```
It took me a moment to understand why this `world_->getObject(id)` call always returns a valid object.

Reply:
Done. Yes, the world object structure is a little awkward.